sdk/go/kms_test.go

```diff
@@ -0,0 +1,157 @@
+		if err != nil {
+			t.Error(err)
+		}
+		signature256, err := client.Sign(keyid, "EH_PAD_NONE", "EH_SHA_256", "EH_DIGEST", base64.StdEncoding.EncodeToString(msg_digest256[:]))
+		if err != nil {
+			t.Error(err)
+		}
+		result256, err := client.Verify(keyid, "EH_PAD_NONE", "EH_SHA_256", "EH_DIGEST", base64.StdEncoding.EncodeToString(msg_digest256[:]), signature256)
+		if result256 != true {
+			t.Error(err)
+		}
+	}
+}
+
+func TestRSASignVerifyWithRaw(t *testing.T) {
+	client, err := NewClient()
+	if err != nil {
+		t.Error(err)
+	}
+	keyspec := []string{"EH_RSA_3072", "EH_RSA_4096", "EH_RSA_2048"}
+	msg := "test for signature"
+	for _, keyspec := range keyspec {
+		keyid, err := client.CreateKey(keyspec, "EH_INTERNAL_KEY", "EH_KEYUSAGE_SIGN_VERIFY")
+		if err != nil {
+			t.Error(err)
+		}
+
+		signature, err := client.Sign(keyid, "EH_RSA_PKCS1", "EH_SHA_256", "EH_RAW", base64.StdEncoding.EncodeToString([]byte(msg)))
+		if err != nil {
+			t.Error(err)
+		}
+		result, err := client.Verify(keyid, "EH_RSA_PKCS1", "EH_SHA_256", "EH_RAW", base64.StdEncoding.EncodeToString([]byte(msg)), signature)
+		if result != true {
+			t.Error(err)
+
+		}
+	}
+}
+
+func TestRSASignVerifyWithDigest(t *testing.T) {
+	client, err := NewClient()
+	if err != nil {
+		t.Error(err)
+	}
+	keyspec := []string{"EH_RSA_3072", "EH_RSA_4096", "EH_RSA_2048"}
+	msg := "test for signature"
+	msg_digest256 := sha256.Sum256([]byte(msg))
+	for _, keyspec := range keyspec {
+		keyid, err := client.CreateKey(keyspec, "EH_INTERNAL_KEY", "EH_KEYUSAGE_SIGN_VERIFY")
+		if err != nil {
+			t.Error(err)
+		}
+		signature256, err := client.Sign(keyid, "EH_RSA_PKCS1", "EH_SHA_256", "EH_DIGEST", base64.StdEncoding.EncodeToString(msg_digest256[:]))
+		if err != nil {
+			t.Error(err)
+		}
+		result256, err := client.Verify(keyid, "EH_RSA_PKCS1", "EH_SHA_256", "EH_DIGEST", base64.StdEncoding.EncodeToString(msg_digest256[:]), signature256)
+		if result256 != true {
+			t.Error(err)
+		}
+	}
+}
+
+func TestSM2SignVerify(t *testing.T) {
+	client, err := NewClient()
+	if err != nil {
+		t.Error(err)
+	}
+	keyid, err := client.CreateKey("EH_SM2", "EH_INTERNAL_KEY", "EH_KEYUSAGE_SIGN_VERIFY")
+	if err != nil {
+		t.Error(err)
+	}
+	msg := "test for signature"
+	msg_digest := sm3.Sm3Sum([]byte(msg))
+	message_type := []string{"EH_RAW", "EH_DIGEST"}
+	for _, message_type := range message_type {
+		signature, err := client.Sign(keyid, "EH_PAD_NONE", "EH_SM3", message_type, base64.StdEncoding.EncodeToString(msg_digest[:]))
+		if err != nil {
+			t.Error(err)
+		}
+		result, err := client.Verify(keyid, "EH_PAD_NONE", "EH_SM3", message_type, base64.StdEncoding.EncodeToString(msg_digest[:]), signature)
+		if result != true {
+			t.Error(err)
+		}
+	}
+}
+
+func TestRSAKeyEncryptDecrypt(t *testing.T) {
+	client, err := NewClient()
+	if err != nil {
+		t.Error(err)
+	}
+	msg := "test for signature"
+	keyspec := []string{"EH_RSA_3072", "EH_RSA_4096", "EH_RSA_2048", "EH_SM2"}
```

Comment:
why include SM2 here?

you can combin SM2 and ECC together, such as  TestAymmetricEncryptDecrypt_SM2_ECC

and also please update the RSA test name to TestAymmetricEncryptDecrypt_RSA